mat/gsvd.go

```diff
@@ -353,15 +368,15 @@ func (gsvd *GSVD) UTo(dst *Dense) *Dense {
 		capCols: c,
 	}
 	dst.Copy(tmp)
-	return dst
 }
 
 // VTo extracts the matrix V from the singular value decomposition, storing
 // the result in-place into dst. V is size p×p.
-// If dst is nil, a new matrix is allocated. The resulting V matrix is returned.
 //
-// VTo will panic if the receiver does not contain a successful factorization.
-func (gsvd *GSVD) VTo(dst *Dense) *Dense {
+// If dst is empty, VTo will resize dst to be p×p. If dst is
+// non-empty, VTo will panic if dst is not p×p. VTo will also
```

Comment:
"When dst is non-empty, VTo will panic if dst is not p×p."

Reply:
Done